### PR TITLE
changing to kraken2

### PR DESCRIPTION
**Pull request by LordRust**
Changes for switching from Kraken 1 to kraken 2.
Versioning nf-core
Plus some cosmetic fixes

### CODE REVIEW

File: main.nf [hunk 1]
```diff
@@ -62,24 +62,23 @@ process kraken {
...
-	est_abundance.py -k ${params.brackendb} -i kraken.rep -o ${id}.kraken
+    kraken2 \\
+			--gzip-compressed \\
+            --db ${params.krakendb} \\
+            --threads ${task.cpus} \\
+            --output ${id}.kraken.out \\
+            --report ${id}.kraken.report \\
+            $read_params
+
+    bracken -d /fs1/jonas/krakendb/krakenstd -r 150 -i ${id}.kraken.report -o ${id}.bracken
```
Comment (bjhall) [inline main.nf]: Should this be ${params.krakendb} rather than hardcoded?

**Review by bjhall — CHANGES_REQUESTED** [review]

**Comment by bjhall** [comment]
You also have a merge conflicts since you reused your old branch without rebasing or merging with master. I can fix the conflict, since it is simple. 

My recommended workflow is to:

1. Create a new branch from master for every pull request:

```
git checkout master && git pull
git checkout -b new-feature-branch
```

2. Do your thing in the new branch.

3. Push and make pull request just like you did with this one.

File: main.nf (hunk 1 above)
Comment (LordRust) [reply main.nf]: Jup, I had it changed, but accidentally not commited the last version of the main.nf script

Comment (bjhall) [reply main.nf]: You can commit and push to this branch and the pull request will update.

Comment (bjhall) [reply main.nf]: Nvm, I changed it.

**Pull request merged**